### PR TITLE
Enable ntruprime sntrup761

### PR DESCRIPTION
**Pull request by loganaden**

### CODE REVIEW

**Review by dstebila — DISMISSED** [review]

File: oqs/Cargo.toml
Comment (thomwiggers) [inline oqs/Cargo.toml:31]:
```suggestion
frodokem = ["oqs-sys/frodokem"]
ntruprime = ["oqs-sys/ntruprime"]
```

**Review by thomwiggers — CHANGES_REQUESTED** [review]
You missed a feature which is required for this to build correctly. Once that's added, this should be ready to be merged.

File: oqs/Cargo.toml
Comment (loganaden) [reply oqs/Cargo.toml:31]: Thanks. PR updated.

**Review by thomwiggers — APPROVED** [review]

**Comment by loganaden** [comment]
It appears that the branch needs to be merged with the latest commit from @thomwiggers before it's merged automatically ?I've done this. Waiting for the next CI run to be allowed.

**Comment by thomwiggers** [comment]
I wish "approve and run" would stick, but alas. Okay, this should finally go through when CI passes now.

**Comment by thomwiggers** [comment]
Thanks :)

**Comment by Sculas** [comment]
I was taking a look at the codebase, and I noticed that NTRU Prime was missing from here:
https://github.com/open-quantum-safe/liboqs-rust/blob/6918c275becb51780ec4761c6974a2843726f3f9/oqs-sys/build.rs#L61-L78
This made me think that the NTRU Prime algorithm would always be disabled, but tests are saying otherwise.
How can it be that NTRU Prime is being linked without it being toggled on? Does it default to being linked?

**Pull request merged**
